crates/wasmtime/src/linker.rs

```diff
@@ -270,6 +271,117 @@ impl Linker {
         Ok(self)
     }
 
+    /// Define automatic instantiations of a [`Module`] in this linker.
```

Comment:
I think this should highlight very clearly that that the major purpose of this method is to transparently handle WASI conventions as well, e.g. "WASI" should be somewhere in this heading.

Reply:
I'm actually trying to de-emphasize "WASI" here. Commands and Reactors are currently defined in a WASI-repository page, but they're not related to WASI APIs, and I expect they'll eventually move out into the linking and/or interface-types specs.